resources/home/dnanexus/calc_custom_coverage.py

```
@@ -28,51 +28,73 @@
-    hs_data = pd.read_csv(hs_metrics_file, sep='\t', header=8, index_col=False,
-        usecols=["coverage_or_base_quality", "high_quality_coverage_count"]
+    hs_data = pd.read_csv(
+        hs_metrics_file,
+        sep="\t",
+        header=8,
+        index_col=False,
+        usecols=["coverage_or_base_quality", "high_quality_coverage_count"],
     )
     # Get the sample name by removing extensions
-    sample_name = file.rstrip('.hsmetrics.tsv')
+    sample_name = file.rstrip(".hsmetrics.tsv")
```

Ruff (0.8.2)

Using `.strip()` with multi-character strings is misleading

(B005)